rsa/Makefile

```diff
@@ -0,0 +1,13 @@
+all:
+	gcc genrsa.c  -L/opt/openssl/lib -lcrypto -o genrsa
```

Comment:
Why /opt/openssl/lib, we have to remove this. And change make file to use Makefile constructs to compiles all c files at ones instead of compiling them individually.